low-level/solution/lr2/support_bundle/sspl_support_bundle

```diff
@@ -0,0 +1,500 @@
+#!/usr/bin/env python3
+
+# CORTX Python common library.
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+import os
+import sys
+import glob
+import time
+import json
+import errno
+import shlex
+import shutil
+import psutil
+import tarfile
+import logging
+import argparse
+import subprocess
+
+from datetime import datetime
+
+from cortx.utils.conf_store import Conf
+from cortx.utils.kv_store import KvStoreFactory
+from framework.base import sspl_constants as sspl_const
+from framework.utils.conf_utils import (
+    Conf, SSPL_CONF, SYSTEM_INFORMATION, SSPL_STATE)
+from framework.utils.ipmi_client import IpmiFactory
+from cortx.utils.process import SimpleProcess
+
+
+# Load bundle request status tracker
+store_type = "json"
+try:
+    bundle_tracker = "/var/cortx/resources/"
```

Comment:
Is this path configurable ?Got from SB framework ?

Reply:
This path will contain a file `bundle_requests.json` which will keep track of support bundle requests. I have moved it to SSPL constants. This is not dependent on support bundle framework.